test/remote/gateways/remote_rapyd_test.rb

```diff
@@ -0,0 +1,156 @@
+require 'test_helper'
+
+class RemoteRapydTest < Test::Unit::TestCase
+  def setup
+    @gateway = RapydGateway.new(fixtures(:rapyd))
+
+    @amount = 100
+    @credit_card = credit_card('4111111111111111', first_name: 'Ryan', last_name: 'Reynolds', month: '12', year: '2035', verification_value: '345')
+    @declined_card = credit_card('4111111111111105')
+    @check = check
+    @options = {
+      type: 'us_visa_card',
+      currency: 'USD'
+    }
+    @ach_options = {
+      type: 'us_ach_bank',
+      currency: 'USD',
+      proof_of_authorization: true,
```

Comment:
Is this field always gonna be `true`?

Reply:
If the merchant is following protocol and getting authorization from their customer prior to processing an ACH then yes, it'll be true. I tested it with `false` and it'll still pass at the gateway level. This is only a required field for ACH payments.